### DISCUSSION TITLE
Maintainers for `thyme` and `true-name`?

### DISCUSSION THREAD

**Discussion by nuttycom** [haskell-github-trust/.github#11] [General]
Hi, there,

Is there a way to identify who is currently acting as maintainer for the `thyme` and `true-name` packages?

I've had an [outstanding pull request](https://github.com/haskell-github-trust/true-name/pulls) for the `true-name` package that has been open for a couple of years at this point. It is currently usable [via nixpkgs](https://github.com/NixOS/nixpkgs/pull/208428/files) but it would be nice to get this published to Hackage.

I'm willing to aid in maintainership for `thyme` and `true-name` if needed.

**Comment by jamesdbrock** [comment]
Hi @nuttycom . I sent you an ownership invitation.  [thyme](https://hackage.haskell.org/package/thyme) and [true-name](https://hackage.haskell.org/package/true-name) both have **haskell-github-trust** maintainers, so if you accept the invitation then you can [publish new versions](https://github.com/haskell-github-trust#how-to-publish-to-hackage) of these packages. Your PR is over a year old has hasn't been reviewed, so I think you should merge it yourself and publish the improvement.

**Reply by nuttycom** [reply]
Thank you!

**Reply by jamesdbrock** [reply]
Thank you for improving these packages!

**Reply by nuttycom** [reply]
I will find someone to review it as well.
